telegram/ext/filters.py

```diff
@@ -1761,6 +1780,46 @@ def filter(self, message: Message) -> bool:
     DELETE_CHAT_PHOTO = _DeleteChatPhoto(name="filters.StatusUpdate.DELETE_CHAT_PHOTO")
     """Messages that contain :attr:`telegram.Message.delete_chat_photo`."""
 
+    class _ForumTopicEdited(MessageFilter):
+        __slots__ = ()
+
+        def filter(self, message: Message) -> bool:
+            return bool(message.forum_topic_edited)
+
+    FORUM_TOPIC_EDITED = _ForumTopicEdited(name="filters.StatusUpdate.FORUM_TOPIC_EDITED")
+    """Messages that contain :attr:`telegram.Message.forum_topic_edited`.
+
+    .. versionadded:: 20.0
+    """
+
+    class _GeneralForumTopicHidden(MessageFilter):
+        __slots__ = ()
```

Comment:
This will give merge conflicts with #3428. Maybe we can just cherry pick 0db57e3d047dc6e9742e5a249a0693cddbce914c into this branch as well?

Reply:
okay, I wanted the diff to be clean, i'll cherry-pick it